docs/sources/setup-grafana/configure-security/configure-authentication/passwordless/index.md

```diff
@@ -0,0 +1,52 @@
+---
+aliases:
+  - ../../../auth/grafana/
+description: Learn how to configure passwordless authentication with magic links in Grafana
+labels:
+  products:
+    - enterprise
+    - oss
+menuTitle: Passwordless auth
+title: Configure passwordless authentication with magic links
+weight: 200
+---
+
+# Passwordless authentication with magic links
+
+The Passwordless integration in Grafana allows your Grafana users to authenticate with a magic link or one-time password (OTP) sent via email.
```

Comment:
"Integration" has a special meaning in Grafana: https://grafana.com/docs/grafana-cloud/monitor-infrastructure/integrations/

```suggestion
Passwordless authentication lets Grafana users authenticate with a magic link or one-time password (OTP) sent via email.
```